docs/include/conf/virtual/welcome-wagon.rst

```diff
@@ -0,0 +1,362 @@
+Hello!
+
+We’re your Welcome Wagon, and we’re glad you’re attending Write the
+Docs! We realize a virtual conference might be totally new to you. This guide will help you navigate it, even if you’ve been to a
+Write the Docs conference before.
+
+Whether this is your first Write the Docs conference, or your first virtual Write the Docs conference, we'll be here to help you make the most of it. We’d love to
+virtually meet you: feel free to reach out in Chat at the virtual
+conference if we can help make your time at the conference easier. You can also email us at {{ shortcode }}@writethedocs.org.
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+Where to get help
+-----------------
+
+Conference Intro and Tour
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Join our our Conference Intro and Tour on Sunday, April 25th at 12:30 pm PST by going to **Sessions** and selecting **Conference Intro and Tour**. Your Welcome Wagon will take you on a tour of the virtual conference platform and show you how to navigate the conference. Learn tips and tricks, meet other attendees, and get to know how the conference works. This session is open to everyone, whether this is your first conference or your fifth.
```

Comment:
These times should be templated too -- are we doing this for Prague as well? I don't know if we have in the past.

Reply:
I think virtual conference tour was new this year in Portland and a great success, so let's assume it will happen for now.